lte/gateway/python/magma/enodebd/devices/experimental/cavium.py

```diff
@@ -79,7 +77,7 @@ def _init_state_map(self) -> None:
             'wait_post_reboot_inform': WaitInformMRebootState(self, when_done='wait_reboot_delay', when_timeout='wait_inform'),
             # The states below are entered when an unexpected message type is
             # received
-            'unexpected_fault': ErrorState(self, inform_transition_target='wait_inform')
+            'unexpected_fault': ErrorState(self)
```

Comment:
revert this line please - we added some remediation logic to this ErrorState state